launch/composition_launch.py

```diff
@@ -6,12 +6,26 @@
 """
 
 import launch
+import os
 from launch_ros.actions import ComposableNodeContainer
 from launch_ros.descriptions import ComposableNode
+from ament_index_python.packages import get_package_share_directory
+
+# Set your camera namespace
+NAMESPACE = "/camera"
```

Comment:
There are several different name styles. I'd prefer if all names were using under_scores style to match the typical ROS practice.